src/hooks/useAppState/reducer.js

```diff
@@ -16,8 +16,8 @@ const reducer = (state, { type, data }) => {
     case types.SET_LIST_ID:
       return { ...state, listId: data.listId };
 
-    case types.SET_ITEMS:
-      return { ...state, items: data.items };
+    case types.SET_STATE:
+      return data.state;
```

Comment:
We now store the whole state in idb, so we need a way to set it all (rather than just the items)